content/tracing/product_specs/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Product Specs
+kind: documentation
+---
+
+{{< whatsnext desc="Enclosed in this section are other articles that may be of interest:">}}
+    {{< nextlink href="/api/?lang=python#tracing" >}} Learn more about our Trace API {{< /nextlink >}}
+    {{< nextlink href="/monitors/monitor_types/apm/" >}} Learn more about monitors with APM {{< /nextlink >}}
+    {{< nextlink href="/graphing/dashboards/widgets/#service-summary" >}} Learn more about the dashboards widgets {{< /nextlink >}}
+    {{< nextlink href="/tracing/product_specs/metrics_namespace" >}} Metrics Namespace{{< /nextlink >}}
+    {{< nextlink href="/tracing/product_specs/trace_sampling_and_storage" >}} Discover our trace sampling and storage policy {{< /nextlink >}}
```

Comment:
I think we can just call this "Trace Sampling and Storage" - same below for "Distributed Tracing Example"

Reply:
👌